configWinVS.bat

```diff
@@ -205,6 +205,9 @@ EXIT /B 0
 :pthread
 ECHO # config pthread
 CD 3rdParty\pthread
+IF NOT EXIST install\\ MKDIR install
+IF NOT EXIST install\\win MKDIR install\\win
+IF NOT EXIST install\\win\\lib MKDIR install\\win\\lib
```

Comment:
This should be moved to `3rdParty\pthread\buildWinVS.bat`.

Reply:
After everything here is working, we can start moving things